Mathlib/Tactic/NthRewrite.lean

```diff
@@ -41,7 +86,54 @@ syntax (name := nthRewriteSeq) "nth_rewrite" (config)? ppSpace num rwRuleSeq (lo
   | _ => throwUnsupportedSyntax
 
 /--
-`nth_rw` is like `nth_rewrite`, but also tries to close the goal by trying `rfl` afterwards.
+`nth_rw` is a variant of `nth_rewrite` that also tries to close the goal by trying `rfl` afterwards.
```

Comment:
I think users will be much more familiar with `rw` than `nth_rewrite`, so I would rephrase in terms of `rw`.
```suggestion
`nth_rw` is a variant of `rw` that only changes the `n`ᵗʰ _occurrence_ of the expression to be
rewritten. Like `rw`, unlike `nth_rewrite` it will try to close the goal by trying `rfl` afterwards.
```

Reply:
This is a very good idea. I'll add it in right away.